### PR TITLE
Move non-Jekyll html files out of _includes folder

### LINKED ISSUE #97: Remove non Jekyll files from the includes directory
- toc.html
- aria-practices.html

Neither are jeckyl includes as far as I ca see from a search. They should be removed. 

The `wai-website` has to link to any required includes and extras like this are confusing

### PR DESCRIPTION
**Pull request by evmiguel**
This issue resolves #97.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       run: |
         bundle install
         npm install -g respec
-        respec2html --src ./_external/aria-practices/aria-practices.html --out ./_includes/aria-practices.html
+        respec2html --src ./_external/aria-practices/aria-practices.html --out ./html/aria-practices.html
```
Comment (SteveALee) [inline .github/workflows/deploy.yml:35]: "html" seems a good as any dir name - we don't use this in the Jekyll config or elsewhere so not confusing.

But note that Jekyll will spider into the folder looking for files with frontmatter to process. to stop that use an `_` prefix for the folder name. See note hidden at foot of [this docs pages](https://jekyllrb.com/docs/structure/).

File: content/about.md [hunk 2]
```diff
@@ -221,7 +221,7 @@ lang: en
       </ul>
     </section>
 
-    <div data-include="common/acknowledgements/funders.html" data-include-replace="true" data-include-id="include-16942119351092688" class="respec-offending-element" title="`data-include` failed: `common/acknowledgements/funders.html` (Failed to fetch)." id="respec-offender-data-include-failed-common-acknowledgements-funders-html-failed-to-fetch"></div>
+    <div data-include="common/acknowledgements/funders.html" data-include-replace="true" data-include-id="include-08728429640203594" class="respec-offending-element" title="`data-include` failed: `common/acknowledgements/funders.html` (Failed to fetch)." id="respec-offender-data-include-failed-common-acknowledgements-funders-html-failed-to-fetch"></div>
```
Comment (SteveALee) [inline content/about.md:224]: can't comment on the  `data-include-id`. I guess is specific to your processes.

File: scripts/pre-build/library/abstractApgContent/abstractApgContent.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ const { fixLinks } = require("./fixLinks");
 const abstractApgContent = async () => {
   const htmlFilePath = path.resolve(
     __dirname,
-    "../../../../_includes/aria-practices.html"
```
Comment (SteveALee) [inline scripts/pre-build/library/abstractApgContent/abstractApgContent.js:11]: don't you just had those long relative paths :D Modern tooling lets you have an alias.

**Review by SteveALee — APPROVED** [review]
A few minor comments

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (evmiguel) [reply .github/workflows/deploy.yml]: @SteveALee I moved the folder to _html.

**Review by richnoah — APPROVED** [review]

**Pull request merged**